app/Observers/DeviceObserver.php

```diff
@@ -38,6 +61,9 @@ public function updated(Device $device)
      */
     public function deleting(Device $device)
     {
+        // delete with NULL Device, so it doesn't get's deleted by daily cleanup scripts
+        Log::event("Device $device->hostname has been removed", null, 'system', 3);
```

Comment:
deleting and deleted are both called when deleting a model.  So you will get two eventlog entries.

Reply:
ok, will remove log event from deleting, but, If you wish we can do it the other way around.
thx for the help.